### PR TITLE
Add editorial moment translations

### LINKED ISSUE #3611: Editorial view of Moment translations
**Is your feature request related to a problem? Please describe.**
We would like to see two lists of moment locales in Storybook, one for devs, which lists all the possible formats and uses. Second, for editorial, which shows formats and information that will be shown on our sites, excluding anything that's not relevant.

**Describe the solution you'd like**
Create a knob, which after changing, will update the [Moment Locales](https://bbc.github.io/simorgh/?path=/story/moment-locales--gujarati-gu) stories in Storybook, with limited view of formats and translations.

## List to display
LL

January
February
March
April
May (long)
June
July
August
September
October
November
December

1 minute in the past
5 minutes in the past
1 hour in the past
5 hours in the past

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
WSFRONTPGS-188

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #3611 

**Overall change:** _Add editorial and developer view of moment translations._

**Code changes:**

- _Add developer and editorial view of moment translations._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/utilities/moment/index.stories.jsx [hunk 1]
```diff
@@ -66,101 +67,122 @@ const NOVEMBER = 1573776000000;
...
-  { what: '1 year in the future', func: locale => moment().locale(locale).add(1, 'years').fromNow() },
-  { what: '5 years in the future', func: locale => moment().locale(locale).add(5, 'years').fromNow() },
+const editorialWhitelist = [
+  'LL',
+  'January',
+  'Febuary',
+  'March',
+  'April',
+  'May',
+  'April',
```
Comment (j-pendlebury) [inline src/app/lib/utilities/moment/index.stories.jsx]: April is here twice 😄

**Review by j-pendlebury — COMMENTED** [review]
Just one thing, otherwise LGTM 👍

File: src/app/lib/utilities/moment/index.stories.jsx [hunk 2]
```diff
@@ -66,101 +67,122 @@ const NOVEMBER = 1573776000000;
...
-  { what: '5 months in the future', func: locale => moment().locale(locale).add(5, 'months').fromNow() },
-  { what: '1 year in the future', func: locale => moment().locale(locale).add(1, 'years').fromNow() },
-  { what: '5 years in the future', func: locale => moment().locale(locale).add(5, 'years').fromNow() },
+const editorialWhitelist = [
+  'LL',
+  'January',
+  'Febuary',
+  'March',
+  'April',
+  'May',
```
Comment (DenisHdz) [inline src/app/lib/utilities/moment/index.stories.jsx]: May is not displayed in the Storybook. You will need to add `May (long)` to the `editorialWhitelist` array to match with `methods`.

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: src/app/lib/utilities/moment/index.stories.jsx (hunk 2 above)
Comment (j-pendlebury) [reply src/app/lib/utilities/moment/index.stories.jsx]: Good spot. February is also not being displayed
image

File: src/app/lib/utilities/moment/index.stories.jsx [hunk 3]
```diff
@@ -66,101 +67,122 @@ const NOVEMBER = 1573776000000;
...
-  { what: '1 day in the future', func: locale => moment().locale(locale).add(1, 'days').fromNow() },
-  { what: '5 days in the future', func: locale => moment().locale(locale).add(5, 'days').fromNow() },
-  { what: '1 month in the future', func: locale => moment().locale(locale).add(1, 'months').fromNow() },
-  { what: '5 months in the future', func: locale => moment().locale(locale).add(5, 'months').fromNow() },
-  { what: '1 year in the future', func: locale => moment().locale(locale).add(1, 'years').fromNow() },
-  { what: '5 years in the future', func: locale => moment().locale(locale).add(5, 'years').fromNow() },
+const editorialWhitelist = [
+  'LL',
+  'January',
+  'Febuary',
```
Comment (DenisHdz) [inline src/app/lib/utilities/moment/index.stories.jsx]:
```suggestion
  'February',
```

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: src/app/lib/utilities/moment/index.stories.jsx (hunk 2 above)
Comment (Bopchy) [reply src/app/lib/utilities/moment/index.stories.jsx]: Fixed the typo, and added `May (long)`. Everything should be displayed now.

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**